merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/generator/ActivityMapperStyler.java

```diff
@@ -16,7 +17,7 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public interface ActivityMapperInstantiator {
```

Comment:
Just for my understanding - what is a `Styler`, and why is that a better name than `Instantiator`?

Reply:
I opted for "Styler" since these classes implement their own logic for different activity type default parameter styles.

"Instantiator" didn't make much sense here because there's not really anything being instantiated, just some logic being chosen depending on the `ActivityTypeDefaultsStyle`. :)

Reply:
Open to any suggestions though – naming has been difficult for me around meta annotation work. A more conventional Java ("enterprise-y" 😉) name would maybe be `ActivityDefaultsStyleMapperStrategy`.

Reply:
Maybe `ActivityMapperMethodMaker`, by analogy with the single method it specifies?

Reply:
Nice! I like that. Just pushed